src/components/date-picker/date-picker-icons.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
+import {properties, stylable} from 'wix-react-tools';
 
-export const CalendarIcon: React.SFC<React.HTMLAttributes<SVGElement>> = props => {
+export const CalendarIcon: React.SFC = props => {
```

Comment:
this change- why?

Reply:
I had an issue with the typing complaining about className, but this wasn't the problem. I reverted it.